docs/diginstroom/sip/1.0/usecases/gigapixel-artwork.md

```diff
@@ -0,0 +1,540 @@
+    |   |   └── dc.xml
+    |   └── preservation
+    |       └── premis.xml
+    │
+    └── representations
+        |── representation_1       # overview with list
+        |   │── mets.xml
+        |   |── data
+        |   │   └── cf9j41p15z_overzichtsopname_metlijst_tiff.tiff
+        |   └──metadata
+        |      |── descriptive    
+        |      |   └── dc.xml   
+        |      └── preservation
+        |          └── premis.xml
+        |── representation_2       # overview without list
+        |   │── mets.xml
+        |   |── data
+        |   │   └── cf9j41p15z_overzichtsopname_zonderlijst_tiff.tiff
+        |   └──metadata
+        |      |── descriptive     
+        |      |   └── dc.xml   
+        |      └── preservation
+        |          └── premis.xml
+        |── representation_3       # composed stitch 
+        |   │── mets.xml
+        |   |── data
+        |   │   └── cf9j41p15z_stitch_bigtiff.tiff
+        |   └──metadata
+        |      |── descriptive   
+        |      |   └── dc.xml         
+        |      └── preservation
+        |          └── premis.xml
+        |── representation_4       # composed stitch in PSB
+        |   │── mets.xml
+        |   |── data
+        |   │   └── cf9j41p15z_stitch_psb.psb
+        |   └──metadata
+        |      |── descriptive   
+        |      |   └── dc.xml         
+        |      └── preservation
+        |          └── premis.xml
+        |── representation_5       # stitch 
+        |   │── mets.xml
+        |   |── data
+        |   |   |── cf9j41p15z_Kolom1_deelopname1_tiff.tiff
+        |   |   |── cf9j41p15z_Kolom1_deelopname2_tiff.tiff
+        |   |   |── ...
+        |   │   └── cf9j41p15z_Kolom5_deelopname6_tiff.tiff`
+        |   └── metadata
+        |       |── descriptive     
+        |       |   └── dc.xml    
+        |       └── preservation
+        |           └── premis.xml
+        └── representation_6       # target 
+           │── mets.xml
+           |── data
+           │   └── cf9j41p15z_target_tiff.tiff
+           └── metadata
+               |── descriptive     
+               |   └── dc.xml    
+               └── preservation
+                   └── premis.xml
+
+```
+
+## The metadata
+
+In total, the SIP contains 6 metadata files:
+
+|`data/metadata/descriptive/dc.xml`| Descriptive metadata about the IE residing at the _package level_ using the DCTERMS metadata schema. |
+|`data/metadata/preservation/premis.xml`| Preservation metadata about the IE residing at the _package level_, including any PREMIS events related to the SIP/package/representations. |
+|`data/representations/representation_*/metadata/preservation/premis.xml`| Preservation metadata about the representation and TIFF files residing at the _representation level_. |
+
+### /data/metadata/descriptive/dc.xml
+
+The `dc.xml` of the package level describes the IE using [the DCTERMS metadata schema](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/).
```

Comment:
Same comment as with the 2D use case: I'm not really sure about the naming conventions of the descriptive metadata files.